### PR TITLE
Logic

### PR DESCRIPTION
**Pull request by MixailE**

### CODE REVIEW

File: src/components/BlockchainList/Status.tsx [hunk 1]
```diff
@@ -1,27 +1,51 @@
 import { PostStatus, PostStatusText } from 'models/PostStatus'
 import { StatusText } from 'components/ui/Text'
+import { Suspense } from 'preact/compat'
+import { useSnapshot } from 'valtio'
 import classnames, { alignItems, display } from 'classnames/tailwind'
+import postIdsStatuses from 'stores/PostIdsStatuses'
 
-const statusContainer = classnames(display('flex'), alignItems('items-center'))
+const statusContainer = () =>
```
Comment (upacyxou) [inline src/components/BlockchainList/Status.tsx:8]: we could use plain classnames here, without a function

File: src/components/BlockchainList/Status.tsx [hunk 2]
```diff
@@ -1,27 +1,51 @@
...
   )
 }
+
+export default function ({ blockchainId }: { blockchainId: number }) {
+  return (
+    <Suspense
+      fallback={
+        <div className={statusContainer()}>
+          <StatusText color={statusColor(PostStatus.pending)}>
+            <span>Loading...</span>
```
Comment (T-Damer) [inline src/components/BlockchainList/Status.tsx:43]: Let's add `tri-dot` here

File: src/components/Cast/TextareaInfo.tsx [hunk 3]
```diff
@@ -36,13 +36,15 @@ export default function () {
...
-            <span>Posting from burner wallet</span>
-            <br />
-            <CharInCircle size={Sizes.Small} char="?" />
-          </span>
-        </Tooltip>
+        <span className={hintWrapper}>
+          <span>Posting from burner wallet</span>
+          <br />
+          <Tooltip position="bottom" fitContainer text={castingHintText}>
+            <div>
```
Comment (T-Damer) [inline src/components/Cast/TextareaInfo.tsx:43]: Do we need this div?

File: src/components/Cast/TextareaInfo.tsx [hunk 4]
```diff
@@ -36,13 +36,15 @@ export default function () {
...
       <AccentText extraSmall color="text-accent">
-        <Tooltip position="bottom" fitContainer text={castingHintText}>
-          <span className={hintWrapper}>
-            <span>Posting from burner wallet</span>
-            <br />
-            <CharInCircle size={Sizes.Small} char="?" />
-          </span>
-        </Tooltip>
+        <span className={hintWrapper}>
+          <span>Posting from burner wallet</span>
```
Comment (T-Damer) [inline src/components/Cast/TextareaInfo.tsx:40]: Do we need `<span>` inside of `<span>`?

File: src/components/navbar/SocialLinks.tsx [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+  space,
+  width,
+} from 'classnames/tailwind'
+
+const socialContainer = classnames(
+  display('inline-flex'),
+  alignItems('items-center'),
+  space('space-x-4')
+)
+const delimiterContainer = classnames(
```
Comment (T-Damer) [inline src/components/navbar/SocialLinks.tsx:20]: Just like in `LoadingWallet`, probably worth extracting or making one component from two

**Review by ohpaul — APPROVED** [review]

**Pull request merged**